CleverTapSDK/CleverTap+InApps.h

```diff
@@ -0,0 +1,65 @@
+@import Foundation;
+#import "CleverTap.h"
+
+@interface CleverTap (InApps)
```

Comment:
Rename `InApps` to `InAppNotification` to follow the existing naming convention.

Reply:
Updated